### DISCUSSION TITLE
Replication to Other RavenDB DB - set expiry metadata

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#13011] [Q&A] [answered]
I want to replicate some of my documents to another RavenDB database that uses document expiry. 

How do you add the expiry metadata to the replication and transform process?

**Comment by garayx** [comment, marked as answer]
Hi

You cannot use [replication](https://ravendb.net/docs/article-page/5.2/csharp/studio/database/tasks/ongoing-tasks/external-replication-task) to modify the documents,

I guess you can use [RavenDB ETL ](https://ravendb.net/docs/article-page/5.2/csharp/server/ongoing-tasks/etl/raven) and add `@expires` poperty to the metadata, ie:

```
this['@metadata']['@expires'] =  "2022-10-10T08:00:00.0000000Z";
loadToUsers(this);
```
